cmd/xl-v1-bucket.go

```diff
@@ -178,7 +178,7 @@ func (xl xlObjects) listBuckets(ctx context.Context) (bucketsInfo []BucketInfo,
 			// should take care of this.
 			var bucketsInfo []BucketInfo
 			for _, volInfo := range volsInfo {
-				if isReservedOrInvalidBucket(volInfo.Name) {
+				if isReservedOrInvalidBucket(volInfo.Name, true) {
```

Comment:
There is no need to do strict name check here. 
Checking reserved word for listing buckets is good enough..

Reply:
If someone changes the bucket names in the backend to have invalid names, even though they are not supposed to touch the backend, then this will not list those buckets. So, this code does not change the existing behavior.